__tests__/Counter-test.tsx

```diff
@@ -0,0 +1,48 @@
+/**
+ * @format
+ */
+
+import React from 'react';
```

Comment:
This is great tasting, but the spec sheet only called out Jest for testing. This is clearing using Jest to set up testing and then using Enzyme to do the actual testing. Jest started out having pretty bad binding to watching changes in react, but it's much better now: https://jestjs.io/docs/en/tutorial-react so much so that I feel that Enzyme is unneeded. Please rewrite these tests using Jest and Snapshots.